src/common/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,69 @@
+import cn from 'classnames';
+import React, {Component} from 'react';
+import ITabsProps from './Tabs.d';
+
+import styles from './tabs.scss';
+
+export default class Tabs extends Component<ITabsProps> {
+	public static defaultProps = {
+		tabs: [
+			{
+				title: '1',
```

Comment:
what is title: 1 ?

Reply:
agree with Eugene, it's seems like a 'id' field with ReactText type.